### PR TITLE
Add Https to mailto to avoid form insecure message

### PR DESCRIPTION
**Pull request by ryanlalchand**

### CODE REVIEW

File: pages/jobs-form.tsx [hunk 1]
```diff
@@ -8,45 +8,122 @@ export default function JobsForm() {
       </h1>
 
       <form
-        className="text-blue"
-        action="mailto:ryanlalchand@gmail.com"
+        className="p-4 flex-row"
+        action="https://mailto:contact@codeyoungstown.com?subject=New Job Listing"
```
Comment (JoeDuncko) [inline pages/jobs-form.tsx:12]: Does that actually work like that? I have no idea

Comment (JoeDuncko) [reply pages/jobs-form.tsx:12]: After doing some googling, I can't find a source that says to add `https://` to mailto links. Do you have something you were referencing?

**Review by JoeDuncko — CHANGES_REQUESTED** [review]
Note that this PR includes changes that are in other PRs, so it's kinda hard to tell what was supposed to be in just this one.

Note comments

File: pages/jobs-form.tsx (hunk 1 above)
Comment (ryanlalchand) [reply pages/jobs-form.tsx:12]: Just some Stack Overflow post as a hail mary. When I pushed this, the Vercel hosted version didn't display the Autofill Danger error

**Review by JoeDuncko — APPROVED** [review]

**Pull request merged**